backend/migrations/00073_add_learning_record_feature_flag.sql

```
@@ -0,0 +1,10 @@
+-- +goose Up
+-- +goose NO TRANSACTION
+ALTER TYPE feature ADD VALUE IF NOT EXISTS 'learning_record';
+
+INSERT INTO public.feature_flags (name, enabled)
+VALUES ('learning_record', FALSE)
+ON CONFLICT (name) DO NOTHING;
+
+-- +goose Down
+DELETE FROM public.feature_flags WHERE name = 'learning_record';
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)